src/Features/ExternalAccess/OmniSharp/Options/OmniSharpEditorConfigOptions.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.ImplementType;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Options;
+
+internal sealed record class OmniSharpEditorConfigOptions
```

Comment:
For the time being these are the only fallback options that have matching O# options